include/stdio.h

```diff
@@ -64,7 +64,7 @@ int remove( const char * filename );
    If there already is a file with the new filename, behaviour is defined by
    the glue code (see functions/_PDCLIB/rename.c).
 */
-int rename( const char * old, const char * new );
+int rename( const char * old, const char * new_name );
```

Comment:
This was already fixed upstream. We shouldn't be fixing these sort of things downstream.

See https://github.com/DevSolar/pdclib/pull/2